ocs_samples/basic_samples/SDS/JavaScript/Angular/src/app/datasrc/datasrc.component.ts

```diff
@@ -592,6 +593,17 @@ export class DatasrcComponent {
       });
   }
 
+  queryTypes() {
+    this.sdsService.getTypes(0, 100, 'Id:*Target*')
```

Comment:
I think in Derek's changes and in this PR the "Target" search value does not match. It looks like we don't create Types with Id values that have the word "target" so the search query probably won't match. Instead, it looks like a value of "Type" replacing "Target" would work. It's possible I missed where the "Target" value comes up. I can check with Derek too since he looked at this Samples repo recently

Reply:
@Annamn61 and I discussed this offline and there are target types created that will match on this query. I didn't see this in the samples. This query string looks good